.github/workflows/adapter-code-coverage.yml

```diff
@@ -0,0 +1,90 @@
+name: Adapter code coverage
+on:
+  pull_request_target:
+    paths: ["adapters/*/*.go"]
```

Comment:
Should we restrict this to just adapters or Do you think we should do it for the entire prebid-server codebase?

Reply:
for initial run lets limit it to just adapters